src/Exporter.cpp

```diff
@@ -0,0 +1,90 @@
+#include <iostream>
+
+#include "Exporter.h"
+
+namespace {
+    const std::string EXPORT_DIR = "exports/";
+    const std::string FORMAT_ID = "obj";
+    const unsigned int VERTEX_COUNT_PER_FACE = 3;
+}
+
+void Exporter::ExportGeometries( const std::vector<Geometry*>& geometries, const std::string filename ) const {
```

Comment:
I don't see how this is exporting multiple geometries.

Reply:
Good catch